Makefile

```diff
@@ -208,7 +208,11 @@ test-ext-dev-e2e: $(OPERATOR_SDK) $(KUSTOMIZE) $(KIND) #HELP Run extension creat
 	test/extension-developer-e2e/setup.sh $(OPERATOR_SDK) $(CONTAINER_RUNTIME) $(KUSTOMIZE) $(KIND) $(KIND_CLUSTER_NAME) $(E2E_REGISTRY_NAMESPACE)
 	go test -count=1 -v ./test/extension-developer-e2e/...
 
-UNIT_TEST_DIRS := $(shell go list ./... | grep -v /test/)
+ENVTEST_VERSION := $(shell go list -m k8s.io/client-go | cut -d" " -f2 | sed 's/^v0\.\([[:digit:]]\{1,\}\)\.[[:digit:]]\{1,\}$$/1.\1.x/')
```

Comment:
We have this value already.
See; https://github.com/operator-framework/operator-controller/blob/7fc18c64660c97c70e4c6704147a746f657543f4/Makefile#L35-L48

Could you please rebase with main and ensure use that instead

Reply:
ah yep, nothing to rebase as it was caused by my error rebase merging. I just removed the dupe in new commit. TY!